styling-and-themes/theme-swatches.md

```diff
@@ -14,76 +14,92 @@ Telerik UI for Blazor comes with three [built-in (base) themes]({%slug general-i
 
 #### In this article:
    * [Basics](#basics)
-   * [Buit-in themes and swatches list](#buit-in-themes-and-swatches-list)
+   * [Built-in themes and swatches list](#built-in-themes-and-swatches-list)
 
 
 ## Basics
 
 The theme swatches are different color variations of the base themes. While they use the same variables as the built-in theme they accompany, they have different color values to deliver a variety of shades for the Telerik components. Any of the base themes and their swatches can be easily customized as desired - read more about that in the [Custom Themes article]({%slug themes-custom%}).
 
-You can find the built-in themes and their swatches in the following resources:
+You can find the built-in themes and all their swatches in the following resources:
 
 * [Sass Themebuilder](https://themebuilder.telerik.com/blazor-ui) - it is a web application that enables you to [create new or customize existing themes]({%slug themes-custom%}). You can use it to explore how certain theme/swatch will affect the appearance of the Telerik UI for Blazor components.
 
 * [kendo-themes repository](https://github.com/telerik/kendo-themes) - the repository for all themes and swatches that are used across Telerik and Kendo web components.
 
-* ZIP archive for Telerik UI for Blazor - the `styles` folder contains the base themes and a some of their most commonly used swatches. The [ZIP Archive]({%slug installation/zip%}) article provides more details on how to get the archive.
+In addition, Telerik UI for Blazor distributes the most commonly used swatches via several other resources. Read more in the [Swatch Distribution]({%slug swatch-distribution%}) article.
 
 
-## Buit-in themes and swatches list
+## Built-in themes and swatches list
 
-Here is a complete list of the base themes, all available swatches and their CDN URLs.
+Here is a complete list of the base themes, all available swatches and their CDN URLs in a dedicated Themes CDN.
 
-* [Default](#default) - These are the mostly ported swatches from our [Sass Themebuilder](https://themebuilder.telerik.com/blazor-ui). Main swatch is the current "Default" theme.
+* [Default](#default) - These are the mostly ported swatches from our [Sass Themebuilder](https://themebuilder.telerik.com/blazor-ui). Main swatch is the [accessibility compliant Ocean Blue swatch]({%slug accessibility-swatch%}) of the "Default" base theme.
```

Comment:
What exactly is a "main" swatch? It looks like we define this nowhere. Does the "main" swatch matter in some way? Also, are we changing the main swatch for the Default theme with this release? Do we need to document a breaking change?

Reply:
Main indeed sounds confusing in this context. The purpose of the sentence is to point which is the initially selected/default swatch in the demos (and in the VS wizard starting as of the May release).
We will not have breaking changes - the current default-main will stay as is (https://github.com/telerik/web-components-planning/issues/45#issuecomment-1057906212). We are just adding an accessibility compliant default theme swatch which will be the initially selected in the demos.